tests/benchmarks/multicore/idle_adc/testcase.yaml

```diff
@@ -0,0 +1,19 @@
+common:
+  sysbuild: true
+  depends_on: adc
+  tags: ci_build ci_tests_benchmarks_multicore adc ppk_power_measure
+
+tests:
+  benchmarks.multicore.idle_adc.nrf54h20dk_cpuapp_cpurad.s2ram:
+    harness: pytest
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    extra_args:
+      - FILE_SUFFIX=s2ram
+      - SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+    harness_config:
+      fixture: ppk_power_measure gpio_loopback
```

Comment:
This is not possible, only single fixture is possible. Thus please gpio_loopback.
Test will be filtered by tag, so they all will execute on HW with ppk_power_measure